containers/ffmpeg-api/skaffold.yaml

```diff
@@ -12,28 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: skaffold/v3
+apiVersion: skaffold/v4beta11
```

Comment:
Is it common practice to use the beta now? Or do we need it for a feature?

Reply:
Scaffold versioning is weird.  Was going off this: https://cloud.google.com/deploy/docs/using-skaffold/select-skaffold#supported_versions

Reply:
Cool, leave it.